pkg/kubectl/cmd/util/helpers.go

```diff
@@ -203,26 +205,29 @@ func StandardErrorMessage(err error) (string, bool) {
 	return "", false
 }
 
-// MultilineError returns a string representing an error that splits sub errors into their own
-// lines. The returned string will end with a newline.
-func MultilineError(prefix string, err error) string {
-	if agg, ok := err.(utilerrors.Aggregate); ok {
+// MultilineErrorString returns a string representing an error that splits sub errors into their own
+// lines. The returned string will end with a newline. The prefix should not end in . or : or \n.
+func MultilineErrorString(err string, subErr error) string {
+	if subErr == nil {
+		return fmt.Sprintf("%s.\n", err)
```

Comment:
I don't recommend making assumptions/requirements about the string you get here. This is used through `CheckErr` with messages that come from various different places (API, commands, etc) and it is quite unpredictable. So either remove the `.` or make it smarter (trim spaces then add the dot if not present in the end, etc).